README.md

```diff
@@ -14,29 +17,57 @@ package main
 
 import (
 	"fmt"
+	"log"
 	"os"
 
-	"github.com/supabase-community/storage-go"
+	storage_go "github.com/supabase-community/storage-go"
 )
 
 func main() {
-	client := storage_go.NewClient("https://abc.supabase.co/storage/v1", "<service-token>", nil)
+	client := storage_go.NewClient("https://<project-reference-id>.supabase.co/storage/v1", "<project-secret-api-key>", nil)
 
-	// Get buckets
-	fmt.Println(client.ListBuckets())
+	// Create a new bucket
+	bucket, berr := client.CreateBucket("bucket-id", storage_go.BucketOptions{Public: true})
 
-	// Upload a file
+	if berr.Error != "" {
+    log.Fatal("error creating bucket, ", berr)
```

Comment:
The spaces here are correct to a go file, but sometimes a markdown file doesn't look well. Maybe changing some spaces will be fine.

Reply:
I'll sort that out, thanks for the feedback!